conductor/lib/common.py

```diff
@@ -471,6 +481,9 @@ def get_environment_config(self):
 
             config_key_name = var_name[len(prefix):].lower()
             environment_config[config_key_name] = self._process_var_value(var_value)
+            
+        if 'CONDUCTOR_API_KEY' in os.environ:
```

Comment:
Instead of adding this special logic here (in the `get_environment_config` method), what do you think about adding it to the class `__init__` https://github.com/AtomicConductor/conductor_client/blob/CT-1451-allow-the-conductor-api-key-to-be-specified-via-an-env-variable/conductor/lib/common.py#L419, e.g. 
```python
        # decode api key value (if it exists)
        combined_config['api_key'] =  combined_config.get('api_key', '').decode('ascii')
```
The benefit of putting it there is that it opens up the possibility for a user to store the api key in the config.yml file (like we do with the rest of our config variables).

Reply:
I had thought about that... but it doesn't need to be base64 encoded if it's in the file...so that option already exists.. but then I guess it breaks consistency (that it always needs to be base64 encoded). Let's chat about it at the same time as your comment above.